app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -179,8 +203,36 @@ public boolean onContextItemSelected(MenuItem item)
     @Override
     public boolean onCreateOptionsMenu(Menu menu)
     {
+        _menu = menu;
+
         getMenuInflater().inflate(R.menu.main_menu, menu);
 
+        SearchManager searchManager = (SearchManager) getSystemService(Context.SEARCH_SERVICE);
+        SearchView searchView = (SearchView) menu.findItem(R.id.action_search).getActionView();
+        searchView.setSearchableInfo(searchManager.getSearchableInfo(getComponentName()));
```

Comment:
searchManager could be null here. Could there be a null check, and if it is null the search view is not setup?